utils/__init__.py

```diff
@@ -0,0 +1,60 @@
+"""
+Utilities for the data_processing module
+"""
+
+from tqdm import tqdm
```

Comment:
```suggestion
from tqdm.auto import tqdm
```

This will give us a nicer progress bar in a notebook.